jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/java/org/jbpm/workbench/ht/client/editors/taskslist/TaskAdminSummaryDataSetMapper.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2017 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jbpm.workbench.ht.client.editors.taskslist;
+
+import java.util.function.BiFunction;
+
+import org.dashbuilder.dataset.DataSet;
+import org.jbpm.workbench.ht.model.TaskSummary;
+
+import static org.jbpm.workbench.common.client.util.DataSetUtils.*;
+import static org.jbpm.workbench.ht.model.TaskDataSetConstants.*;
+
+public class TaskAdminSummaryDataSetMapper implements BiFunction<DataSet, Integer, TaskSummary> {
```

Comment:
Do we really need two separate mappers here?
Both mappers are returning the same type (_TaskSummary_) and there's a difference in only one additional setter and the boolean value _isForAdmin_ method is taking as an argument.

So what do you think, for example, about having mapper which doesn't implement _BiFunction_, but rather exposes custom _apply_ method which would take three arguments: 
```
apply(final DataSet dataSet, 
      final Integer row, 
      final boolean isForAdmin)
```
@cristianonicolai Wdyt?

Reply:
@bsremac I had only one mapper at first, but @cristianonicolai suggested having two separate mappers.

Reply:
@mcivantos-tribalyte I suppose the reason is having the consistent implementation of all mapper classes, I can't see any other reason to be honest, since the "functional" part of the mapper implementation is not even being used.

Reply:
@mcivantos-tribalyte @bsremac I agree, the mapper implementation isn't significantly different to justify a separated class. I guess I initially thought of this without checking the columns that are used in each case. So my primary concern is to not repeat this entire logic for both scenarios. 
Perhaps something as simple as this could work?
```java
.errorCount(HUMAN_TASKS_WITH_ADMIN_DATASET.equals(dataSet.getUUID()) ? getColumnIntValue(dataSet,
                                                                                                             COLUMN_ERROR_COUNT,
                                                                                                             row) : null)
                    .isForAdmin(HUMAN_TASKS_WITH_ADMIN_DATASET.equals(dataSet.getUUID()))
´´´